Configure

```diff
@@ -144,6 +144,7 @@ my @gcc_devteam_warn = qw(
 #       -Wunused-macros -- no, too tricky for BN and _XOPEN_SOURCE etc
 #       -Wextended-offsetof -- no, needed in CMS ASN1 code
 my @clang_devteam_warn = qw(
+    -Wno-unknown-warning-option
```

Comment:
Triple Ups! How could miss that?  

OOH, your discovery makes this pr superfluous. OTOH, I would like to keep it because of the commit message, and place the flag at the beginning due to its importance and for consistency with 1.0.2 

What do you think?

Reply:
ah, a joke, yes ? :-)

Reply:
> ah, a joke, yes ? :-)

Ok, I hope the now commit message does not make it look like a joke anymore and justifies this 'trivial' change:

commit 48b8a832d01b27d40697cba8d9a169be0f365c42 
Author: Dr. Matthias St. Pierre <Matthias.St.Pierre@ncp-e.com>
Date:   Tue Jul 23 20:54:03 2019 +0200

    Configure: clang: move -Wno-unknown-warning-option to the front
    
    While gcc ignores unknown options of the type '-Wno-xxx', clang by default issues
    a warning [-Wunknown-warning-option] (see [3]), which together with '-Werror'
    causes the build to fail. This turned out to be a problem on the 1.0.2 stable branch
    in the case of the '-Wextended-offsetof' option, which was removed in version 6.0.0,
    but needs to be kept here in order to support older clang versions, too (see #9446).
    
    Incidentally, master and 1.1.1 branch already contained the -Wno-unknown-warning-option
    option. Due to its special role and its importance, this commit adds an explaining
    commit message and moves the option to the front.
    
    [extended tests]